### PR TITLE
[SD-4188] (Enabling e2e) can navigate/filter subject field and search…

### PR DESCRIPTION
**Pull request by marwoodandrew**
… by selected subject item

### CODE REVIEW

**Comment by akintolga** [comment]
:+1:

File: spec/search_spec.js [hunk 1]
```diff
@@ -94,9 +95,10 @@ describe('search', function() {
         expect(globalSearch.getItems().count()).toBeGreaterThan(0);
 
         // now preview first item on search list and expect item contains selected subject
-        globalSearch.itemClick(0);
-        monitoring.tabAction('metadata');
-        expect(globalSearch.getItemSubjectContains()).toContain('archaeology');
+        // SUBJECT IS NO LONGER IN THE MATADATA TAB OF PREVIEW
```
Comment (ioanpocol) [inline spec/search_spec.js]: I think we can delete those commented lines

**Comment by ioanpocol** [comment]
:+1:

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**